app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: 'posts_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'users_id'
+
+  after_save :increment_comment_couter
+
+  def increment_comment_couter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] The same here, since we are calling this method from the callback function, then ensure this method is private